libs/langchain-google-common/src/utils/common.ts

```diff
@@ -1,26 +1,40 @@
-import type { GoogleAIModelParams, GoogleLLMModelFamily } from "../types.js";
+import type {
+  GoogleAIBaseLanguageModelCallOptions,
+  GoogleAIModelParams,
+  GoogleAIModelRequestParams,
+  GoogleLLMModelFamily,
+} from "../types.js";
 import { isModelGemini, validateGeminiParams } from "./gemini.js";
 
 export function copyAIModelParams(
-  params: GoogleAIModelParams | undefined
-): GoogleAIModelParams {
-  return copyAIModelParamsInto(params, {});
+  params: GoogleAIModelParams | undefined,
+  options: GoogleAIBaseLanguageModelCallOptions | undefined
+): GoogleAIModelRequestParams {
+  return copyAIModelParamsInto(params, options, {});
 }
 
 export function copyAIModelParamsInto(
   params: GoogleAIModelParams | undefined,
+  options: GoogleAIBaseLanguageModelCallOptions | undefined,
```

Comment:
Why not make this a third param instead of adding it into the middle?

Reply:
The function is only used internally with the last parameter being the target object.
I could have restructured it to take a variable number of parameters and use them to determine the order that the object should be copied in... But I'm not sure that was worth it.
One could make the argument that the first two parameters should be swapped I suppose... But I think we're over-thinking this now.